frontend/src/js/components/settings/PlanSelection.tsx

```diff
@@ -0,0 +1,124 @@
+      cursor: 'default',
+      boxShadow: 'none'
+    }
+  }
+}));
+const canUpgrade = (plan: AvailablePlans, currentPlan: AvailablePlans) => Object.keys(PLANS).indexOf(plan) > Object.keys(PLANS).indexOf(currentPlan);
+
+const isPlanDisabled = (item: Plan, currentPlan: AvailablePlans, isTrial: boolean, isUpgradePending: Partial<{ plan: AvailablePlans }>) => {
+  return (isUpgradePending.plan && !canUpgrade(item.id, isUpgradePending.plan)) || (!isTrial && !canUpgrade(item.id, currentPlan));
+};
+const isPlanMuted = (item: Plan, currentPlan: AvailablePlans, isTrial: boolean, isUpgradePending: Partial<{ plan: AvailablePlans }>) => {
+  return (
+    isPlanDisabled(item, currentPlan, isTrial, isUpgradePending) &&
+    (item.id !== currentPlan || isTrial) &&
+    !(isUpgradePending.plan && isUpgradePending.plan === item.id)
+  );
+};
+export const PlanSelection = ({ orgId, currentPlan = PLANS.os.id, isTrial, setUpdatedPlan, updatedPlan }) => {
+  const { classes } = useStyles();
+  const onPlanSelect = (plan: AvailablePlans) => (isTrial || canUpgrade(plan, currentPlan) ? setUpdatedPlan(plan) : undefined);
+  const isUpgradePending = isUpgrade(orgId);
+
+  useEffect(() => {
+    if (!isEmpty(isUpgradePending) && currentPlan === isUpgradePending.plan) {
+      clearLocalStorageEntry(orgId, isUpgradePending.plan);
+    }
+  }, [currentPlan, isUpgradePending, orgId]);
+  const getPlanButtonLabel = (planId: AvailablePlans) => {
+    if (!isEmpty(isUpgradePending) && isUpgradePending.plan === planId) return 'pending';
+    if (isTrial) return planId === PLANS.enterprise.id ? 'contact us' : 'subscribe';
+    if (currentPlan === planId) return 'current plan';
+    return planId === PLANS.enterprise.id ? 'contact us' : 'upgrade';
+  };
+
+  return (
+    <>
+      <p className="margin-top">Your current plan: {isTrial ? ' Free trial' : `Mender ${PLANS[currentPlan].name}`}</p>
+      <div className="flexbox margin-bottom-small">
+        {Object.values(PLANS).map(item => (
+          <div
+            key={item.id}
+            className={`planPanel ${classes.planPanel} ${updatedPlan === item.id ? 'active' : ''} ${isPlanMuted(item, currentPlan, isTrial, isUpgradePending) ? 'muted' : ''}`}
+          >
+            <div>
+              <p className="margin-none">Mender {item.name}</p>
+              <h2>{item.price}</h2>
+              <div className="margin-bottom">
+                <div className="flexbox align-center">
+                  <DeveloperBoardIcon fontSize="medium" className={classes.planIcon} />
+                  <div className="margin-left-x-small">
+                    {item.deviceCount}
+                    {item.id === PLANS.enterprise.id ? '' : '*'}
+                  </div>
+                </div>
+              </div>
+              <ul className="unstyled">
+                {item.features.map((feature, index) => (
+                  <li className={`flexbox margin-bottom-x-small ${classes.planFeature}`} key={`${item.id}-feature-${index}`}>
```

Comment:
```suggestion
                  <li className="flexbox margin-bottom-x-small slightly-smaller" key={`${item.id}-feature-${index}`}>
```
to align with the rest of the UI